### ISSUE TITLE
help request: syslog plugin / attribute 'name' confusion

### ISSUE THREAD

**Issue by Wasta-Geek** [apache/apisix#10474]
### Description

Hi

I'm kind of new to Apisix but I wanted to use the syslog plugin and I had trouble to understand the meaning of the plugin key 'name'.
My original need was to modify what would correspond to the 'tag' part (apisix[pid]) in the syslog message and I thought it would correspond to the plugin attribute 'name'.

But this attribute seems to be ignored as I couldn't find it in my remote logs. (other plugin attributes would get applied correctly so it seems not to be a config problem on my side)
I tried to see it via a tcpdump what's been received and I couldn't retrieves the value anywhere, I also tried to grep 'name' in the LUA syslog plugin files and I couldn't see a reference to this.

So I wanted to know what is the usage / impact of this attribute regarding the syslog plugin ?

Thanks for your help, best regards.

### Environment

- APISIX version (run `apisix version`): 3.6.0
- Operating system (run `uname -a`):Linux 5.10.0-25-amd64 #1 SMP Debian 5.10.191-1 (2023-08-16) x86_64 GNU/Linux
- OpenResty / Nginx version (run `openresty -V` or `nginx -V`): nginx/1.18.0
- etcd version, if relevant (run `curl http://127.0.0.1:9090/v1/server_info`): 
- APISIX Dashboard version, if relevant:
- Plugin runner version, for issues related to plugin runners:
- LuaRocks version, for installation issues (run `luarocks --version`):

**Comment by kayx23** [comment]
APISIX loggers [send logs in batches](https://apisix.apache.org/docs/apisix/batch-processor). The name here is the ID of the batch processor for the plugin, which conveniently inherits the plugin's name.

However I noticed the value is hardcoded in, which means what you configure actually doesn't matter. I opened a PR to removed the `name` from attributes for `kafka-logger` here: https://github.com/apache/apisix/pull/10476 

If I'm not wrong on this, we should remove name attribute from the doc for all loggers that hard code the value in.

For sys log specifically, the value is hard coded here:

https://github.com/apache/apisix/blob/e1503dcb2b6220303c6023d7f0fe5e591efecd8f/apisix/plugins/syslog/init.lua#L26

You won't see this value in the actual logs.
